integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringinstallation/CoupleDeviceSteps.java

```diff
@@ -33,53 +33,58 @@ public class CoupleDeviceSteps extends AbstractSmartMeteringSteps {
 
   @Autowired private SmartMeteringInstallationClient smartMeteringInstallationClient;
 
-  @When("^the Couple G-meter \"([^\"]*)\" request is received for E-meter \"([^\"]*)\"$")
-  public void theCoupleGMeterRequestIsReceivedForEMeter(final String gasMeter, final String eMeter)
+  @When("^the Couple M-Bus device \"([^\"]*)\" request is received for E-meter \"([^\"]*)\"$")
+  public void theCoupleMBusDeviceRequestIsReceivedForEMeter(
+      final String mBusDeviceIdentification, final String eMeter)
       throws WebServiceSecurityException {
 
     final CoupleMbusDeviceRequest request =
-        CoupleMbusDeviceRequestFactory.forGatewayMbusDevice(eMeter, gasMeter);
+        CoupleMbusDeviceRequestFactory.forGatewayMbusDevice(eMeter, mBusDeviceIdentification);
     final CoupleMbusDeviceAsyncResponse asyncResponse =
         this.smartMeteringInstallationClient.coupleMbusDevice(request);
 
     this.checkAndSaveCorrelationId(asyncResponse.getCorrelationUid());
   }
 
-  @When("^the Couple G-meter \"([^\"]*)\" request is received for E-meter \"([^\"]*)\" with force$")
-  public void theCoupleGMeterRequestIsReceivedForEMeterWithForce(
-      final String gasMeter, final String eMeter) throws WebServiceSecurityException {
+  @When(
+      "^the Couple M-Bus device \"([^\"]*)\" request is received for E-meter \"([^\"]*)\" with force$")
+  public void theCoupleMBusDeviceRequestIsReceivedForEMeterWithForce(
+      final String mBusDeviceIdentification, final String eMeter)
+      throws WebServiceSecurityException {
 
     final CoupleMbusDeviceRequest request =
-        CoupleMbusDeviceRequestFactory.forGatewayMbusDeviceWithForce(eMeter, gasMeter);
+        CoupleMbusDeviceRequestFactory.forGatewayMbusDeviceWithForce(
+            eMeter, mBusDeviceIdentification);
     final CoupleMbusDeviceAsyncResponse asyncResponse =
         this.smartMeteringInstallationClient.coupleMbusDevice(request);
 
     this.checkAndSaveCorrelationId(asyncResponse.getCorrelationUid());
   }
 
   @When(
-      "^the Couple G-meter \"([^\"]*)\" to E-meter \"([^\"]*)\" request is received for an unknown gateway$")
-  public void theCoupleGMeterToEMeterRequestOnChannelIsReceivedForAnUnknownGateway(
-      final String gasMeter, final String eMeter) throws WebServiceSecurityException {
-
-    final CoupleMbusDeviceRequest request =
-        CoupleMbusDeviceRequestFactory.forGatewayMbusDevice(eMeter, gasMeter);
-
-    try {
-      this.smartMeteringInstallationClient.coupleMbusDevice(request);
-      Assertions.fail("A SoapFaultClientException should be thrown");
-    } catch (final SoapFaultClientException e) {
-      ScenarioContext.current().put(PlatformKeys.RESPONSE, e);
-    }
+      "^the Couple M-Bus device \"([^\"]*)\" to E-meter \"([^\"]*)\" request is received for an unknown gateway$")
+  public void theCoupleMBusDeviceToEMeterRequestOnChannelIsReceivedForAnUnknownGateway(
+      final String mBusDeviceIdentification, final String eMeter)
```

Comment:
nu gasMeter mBusDeviceIdentification is geworden, zou ik het wel mooi vinden om bij de eMeter ook aan te geven dat het de deviceIdentification is, dus eMeterDeviceIdentification. Dit zit op meer plekken.

Reply:
Inderdaad, aangepast